.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    name: Go ${{ matrix.go-version }} - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: ["1.15"]
```

Comment:
Tip is not available using `setup-go`, see https://github.com/actions/setup-go/issues/21